packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js

```diff
@@ -12,6 +12,7 @@ import TaskHelpButton from './components/TaskHelpButton'
 import { default as TaskNavButtons } from './components/TaskNavButtons'
 
 function storeMapper (stores) {
+  const { isActive } = stores.classifierStore.feedback
```

Comment:
I think this can be removed. It's not used anywhere.

Reply:
This is temporary, but needed, otherwise the FeedbackStore never runs afterAttach, and therefore never runs either observer.

Reply:
I thought this would run when the feedback store is created and attached to the root in `RootStore.js` but apparently not? 🤷‍♂️

Reply:
I thought so too, def spent some time scratching head on this. I added comment in Tasks.js. Once UI is added then this line won't be necessary.